test/integration/supported/client-execute-tests.js

```diff
@@ -1303,7 +1302,8 @@ describe("Client @SERVER_API", function () {
                     },
                 );
             });
-            vit(
+            // TODO: Add type guessing for UDT and Tuple
+            /* vit(
```

Comment:
:question: Why isn't this implemented in this PR yet?
:question: When is it going to be implemented?
:wrench: If there's no issue for this missing feature yet, please open it!